src/bin/menu_bar.rs

```diff
@@ -97,6 +102,14 @@ fn build_ui(application: &gtk::Application) {
         window.destroy();
     }));
 
+    quit.add_accelerator(
```

Comment:
This might not seem easy to understand for people who don't know GTK. Can you add a comment explaining that this control is expecting a "CTRL+Q" or a "CMD+Q" please?

Reply:
I have changed the code so it does actually use `Ctrl+Q` on Windows and Linux, and `Cmd+Q` on macOS. Helpfully, when you actually run the example, the key combo required is displayed alongside the menu item's name by GTK .